src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -20,15 +20,48 @@
 
 namespace Stratis.Bitcoin.Features.Miner
 {
+    /// <summary>
+    /// <see cref="PosMinting"/> is used in order to generate new blocks. It involves a sort of lottery, similar to proof-of-work,
+    /// but the chances of winning this lottery is proportional to how many coins you are staking, not on hashing power.
+    /// </summary>
+    /// <remarks>
+    /// Staking is attempted only if the node is fully synchronized and connected to the network.
+    /// If not it will wait  till node is synced. Only transactions that were confirmed at least
+    /// <see cref="PosConsensusOptions.StakeMinConfirmations"/> blocks ago are eligible for staking.
+    /// </remarks>
+    /// <remarks>
+    /// The overall process for "attempting" to mine a PoS block looks like this:
+    /// 1) Create new block with transactions from <see cref="TxMempool"/>
```

Comment:
use <list> - see e.g. PullerDownloadAssignmentsTest how this tag is used

+ missing dot here

and I tend to avoid mentioning concrete implementation of something when it is not needed. So you say here `from TxMempool`, but at this point I'm not sure the reader appreciates this information that it is actually TxMempool class, I think the reader would be happy with just `from mempool`.